src/toast/styles.css

```diff
@@ -0,0 +1,86 @@
+.toast {
```

Comment:
Is there any way we can utilize existing Clarity classnames?  For example, there should already be classnames that we use for the background-color in other components.  I know this isn't a standard VMWare Clarity component, but it would be great if we stayed as closely aligned as possible to the current tool set (in terms of styling).

Reply:
Updated code.